js/ccf-app/src/crypto.ts

```diff
@@ -27,6 +27,11 @@ export const generateAesKey = ccf.generateAesKey;
  */
 export const generateRsaKeyPair = ccf.generateRsaKeyPair;
 
+/**
```

Comment:
This change requires a new entry in the changelog.

Reply:
So it does.